.travis.yml

```diff
@@ -21,7 +21,7 @@ services:
 
 install:
   - npm install -g codecov
-  - npm ci
+  - npm install --package-lock-only
```

Comment:
Err, oops:
> The --package-lock-only argument will only update the package-lock.json, instead of checking node_modules and downloading dependencies.

I didn't double check what this actually did. Let's just try it with `install` again.